gnmi_cli_py/py_gnmicli.py

```diff
@@ -156,7 +158,7 @@ def _create_parser():
                       help='subscription mode [0=TARGET_DEFINED, 1=ON_CHANGE, 2=SAMPLE]')
   parser.add_argument('--update_count', default=0, type=int, help='Max number of streaming updates to receive. 0 means no limit.')
   parser.add_argument('--subscribe_mode', default=0, type=int, help='[0=STREAM, 1=ONCE, 2=POLL]')
-  parser.add_argument('--encoding', default=0, type=int, help='[0=JSON, 1=BYTES, 2=PROTO, 3=ASCII, 4=JSON_IETF]')
+  parser.add_argument('--encoding', default=4, type=int, help='[0=JSON, 1=BYTES, 2=PROTO, 3=ASCII, 4=JSON_IETF]')
```

Comment:
>4 [](http://example.com/codeflow?start=44&length=1)

This is a valid bugfix on its own. Suggest separate into a standalone commit, not to mix with proto feature.

Reply:
This change is not necessary, so I have reverted it.